t/Util.pm

```diff
@@ -698,4 +699,36 @@ sub get_tracer {
     return $read_trace;
 }
 
+sub run_picotls_client {
+    my($opts) = @_;
+    my $port = $opts->{port}; # required
+    my $host = $opts->{host} // '127.0.0.1';
+    my $path = $opts->{path} // '/';
+    my $cli_opts = $opts->{opts} // '';
+    my $connection = $opts->{keep_alive} ? "keep-alive" : "close";
+
+    my $cli = bindir() . "/picotls/cli";
+    die "picotls-cli ($cli) not found" unless -e $cli;
```

Comment:
I do not think we need to add this check, when `open` below would emit the same message if the executable does not exist?